extractor/src/main.py

```diff
@@ -191,20 +211,13 @@ def process_event(msg):
 
         try:
             analysis = analyzer.load_cached_analysis(batch_id, resource_id)
-
             df = extract_resource(analysis, primary_key_values)
-            batch_size = df.count()
-            logger.info(
-                f"Batch size is {batch_size} for resource type {analysis.definition_id}",
-                extra={"resource_id": resource_id},
-            )
```

Comment:
It was nice to have a log telling us how many resources will be extracted.
I know it may not be river's job to do that though... What do you think?

Reply:
Since we use yield_per, does not count == CHUNK_SIZE here?

Reply:
(the log was useful indeed)
it's true that I don't know exactly how it behaves now that we use yield_per.. 
but what if the dataframe is smaller that CHUNK_SIZE for instance ?

Reply:
It works with the yield_per as well, `.count()` actually executes another sql query

Reply:
We now have a counter in Redis telling us what has been extracted during a batch. I don't know if another roundtrip to the db is necessary.